models/types/person.sql

```diff
@@ -0,0 +1,25 @@
+{{
+    config(
+        materialized = 'incremental',
+        indexes=[
+            {'columns': ['"@timestamp"'], 'type': 'brin'},
+            {'columns': ['"patient_id"'], 'type': 'hash'},
+        ]
+    )
+}}
+
+SELECT
+    doc->>'name' AS name,
+    doc->>'date_of_birth' AS date_of_birth,
+    doc->>'phone' AS phone,
+    doc->>'sex' AS sex,
+    doc->>'reported_date' AS reported_date,
+    doc->>'patient_id' AS patient_id,
+    *
+FROM {{ ref('couchdb') }}
+WHERE
+    doc->>'type' = 'person'
+
+{% if is_incremental() %}
+    AND COALESCE("@timestamp" > (SELECT MAX("@timestamp") FROM {{ this }}), True)
```

Comment:
Is '@timestamp' a DBT thing? I've tried some quick googling but wasn't able to find anything useful.
Does it represent the current time or the time of the last run or both?

Reply:
It comes from https://www.elastic.co/guide/en/logstash/current/plugins-inputs-couchdb_changes.html which is being used in https://github.com/medic/cht-sync/pull/1